server/game/drawcard.js

```diff
@@ -331,6 +331,7 @@ class DrawCard extends BaseCard {
         this.power = 0;
         this.wasAmbush = false;
         this.inChallenge = false;
+        this.new = false;
```

Comment:
As long as we're doing this, might as well reset the other UI flags. `resetForChallenge()` will clear stealth info, `clearDanger()` will clear the inDanger and saved flags.